jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt

```diff
@@ -109,7 +110,16 @@ enum class SourceOfEntry {
     CODEWHISPERER,
     EXPLORER,
     FIRST_STARTUP,
-    UNKNOWN
+    UNKNOWN;
+    override fun toString(): String {
+        val value = this.name.lowercase()
+        return if(value.contains("_")){
+            //convert to camelCase
```

Comment:
providing inline example as part of the comment would be helpful for readers

Reply:
test?

Reply:
Modified the comment and added tests